modules/videojsVideoProvider.js

```diff
@@ -25,29 +27,82 @@ export function VideojsProvider(config, videojs_, adState_, timeState_, callback
 
     // returns the player if it exists, or attempts to instantiate a new one
     player = videojs(divId, playerConfig, function() {
-        // callback runs in both cases
+      // callback runs in both cases
     });
+    // TODO: make sure ortb gets called in integration example
+    // currently testing with a hacky solution by hooking it to window
+    // window.ortb = this.getOrtbParams
   }
 
   function getId() {
     return divId;
   }
 
   function getOrtbParams() {
+    if (!player) {
+      return null;
+    }
+
+    let playBackMethod = PLAYBACK_METHODS.CLICK_TO_PLAY
+    const isMuted = player.muted() || player.autoplay() === 'muted'; // todo autoplayAdsMuted only applies to preRoll
+    if (player.autoplay()) {
```

Comment:
what does `autoplay()` return ? In the line above you compare it to a string. In this line you use it as a condition
I like truthy conditions, as long as it can be falsy.